### ISSUE TITLE
Is instance norm layer or batch norm layer necessary for PatchNCE Loss ？

### ISSUE THREAD

**Issue by laulampaul** [taesungp/contrastive-unpaired-translation#55]
I notice that in your codes, you extract the feature maps from the instance norm layer. （4,8,12,16）
what happen if removing the instance norm layer ? 
Does patchnce loss function convergences in this way ?

**Comment by Geziii** [comment]
I have the same question. You also calculated the loss of the output of the Reflection Pad layer. Is there any reason to do this?
